src/test/functional/cypress/integration/feature/admin/jobCategories.spec.js

```diff
@@ -0,0 +1,120 @@
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+describe('Admin - Job Category', function () {
+  beforeEach(function () {
+    cy.task('db:reset');
+    cy.fixture('viewport').then(({HD}) => {
+      cy.viewport(HD.width, HD.height);
+    });
+    cy.fixture('chars').as('strings');
+    cy.intercept('GET', '**/api/v2/admin/job-categories?*').as(
+      'getJobCategories',
+    );
+    cy.intercept('POST', '**/api/v2/admin/job-categories').as(
+      'postJobCategories',
+    );
+    cy.intercept('PUT', '**/api/v2/admin/job-categories/*').as(
+      'updateJobCategories',
+    );
+    cy.intercept('DELETE', '**/api/v2/admin/job-categories').as(
+      'deleteJobCategories',
+    );
+    cy.fixture('user').then(({admin}) => {
+      this.user = admin;
+    });
+  });
+
+  // Read
+  describe('list job Categories', function () {
+    it('job title list is loaded', function () {
+      cy.loginTo(this.user, '/admin/jobCategory');
+      cy.wait('@getJobCategories');
+      cy.getOXD('numRecords').contains('(9) Records Found');
+    });
+  });
+  // Create
+  describe('Add job Category', function () {
+    it('add job category', function () {
+      cy.loginTo(this.user, '/admin/saveJobCategory');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Name').type(this.strings.chars50.text);
+        cy.getOXD('button').contains('Save').click();
+      });
+      cy.wait('@postJobCategories');
+      cy.toast('success', 'Successfully Saved');
+    });
+    it('Job Category form validations', function () {
+      cy.loginTo(this.user, '/admin/saveJobCategory');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Name')
+          .type(this.strings.chars100.text)
+          .isInvalid('Should not exceed 50 characters');
+        cy.getOXDInput('Name').setValue('').isInvalid('Required');
+        cy.getOXDInput('Name')
+          .type('Craft Workers')
+          .isInvalid('Already exists');
+      });
+    });
+    it('add a job category and click cancel', function () {
+      cy.loginTo(this.user, '/admin/saveJobCategory');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Name').type(this.strings.chars50.text);
+        cy.getOXD('button').contains('Cancel').click();
+      });
+      cy.wait('@getJobCategories');
+      cy.getOXD('pageTitle').contains('Job Categories');
+    });
+  });
+  //Update
+  describe('Update job Category', function () {
+    it('Edit job category', function () {
+      cy.loginTo(this.user, '/admin/saveJobCategory/1');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Name').clear().type(this.strings.chars50.text);
+        cy.getOXD('button').contains('Save').click();
+      });
+      cy.wait('@updateJobCategories');
+      cy.toast('success', 'Successfully Updated');
+    });
+  });
+  //Delete
```

Comment:
```suggestion

  //Delete
```
keep new line between describe blocks and test blocks, it's readable then